exercise/views.py

```diff
@@ -379,9 +379,24 @@ def get_common_objects(self):
                     'name': name,
                     'content': response.text,
                     'html': 'text/html' in response.headers.get('Content-Type'),
+                    'data': response.content,
```

Comment:
`data` can be removed if `content` is used in the `get` method instead of `data`.